README.md

```diff
@@ -12,6 +12,19 @@ itself.
 
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
+## Compatibility matrix
+
+|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|---------------------|----------------|----------------|----------------|----------------|
+| **Dashboard 1.4**   | ✓              | ✕              | ✕              | ✕              |
+| **Dashboard 1.5**   | ✕              | ✓              | ✕              | ✕              |
+| **Dashboard 1.6**   | ✕              | ✕              | ✓              | ?              |
+| **Dashboard HEAD**  | ✕              | ✕              | ?              | ✓              |
+
+- `✓` All the functionalities are supported with given version.
+- `?` Some of the functionalities might not be working.
```

Comment:
The main difference is that first declares that it is supported (by us) version of kubernetes core and we guarantee that it will work and if anything does not work then we will fix it.

`?` on the other hand states that due to version differences some functionalities might not work correctly in dashboard and we will not fix that.